repose-aggregator/artifacts/valve/src/main/scala/org/openrepose/valve/ReposeJettyServer.scala

```diff
@@ -128,9 +128,7 @@ class ReposeJettyServer(val clusterId: String,
           }
         }
 
-        Option(ssl.isTlsRenegotiationAllowed).foreach { tls =>
-          cf.setRenegotiationAllowed(tls)
-        }
+        cf.setRenegotiationAllowed(Option(ssl.isTlsRenegotiationAllowed).getOrElse[java.lang.Boolean](java.lang.Boolean.FALSE))
```

Comment:
Weird.

After seeing what you tried, I tried something myself. It worked.

```
cf.setRenegotiationAllowed(Option(ssl.isTlsRenegotiationAllowed).getOrElse(java.lang.Boolean.FALSE).booleanValue())
```

So the Option(java.lang.Boolean).getOrElse(java.lang.Boolean) works and returns a java.lang.Boolean. But `setRenegotiationAllowed` takes a `boolean`, the Java primitive. So Scala doesn't know how to convert `java.lang.Boolean` to boolean, I guess. Scala must do some magic to make a `java.lang.Boolean` into a Scala `Boolean` which provides the `booleanValue` method.

So yeah, pick your favorite. I really don't care about much, but this was an interesting discovery!